src/Provider/UnitTests/Provider.Web/Orchestrators/ApplicationReviewsOrchestratorTests/ApplicationReviewsOrchestratorTests.cs

```diff
@@ -47,11 +47,11 @@ public async Task GetApplicationReviewsToUnsuccessfulViewModelAsync_ReturnsViewM
 
             _vacancyClient.Setup(x => x.GetVacancyAsync(routeModel.VacancyId.GetValueOrDefault()))
                 .ReturnsAsync(vacancy);
-            _vacancyClient.Setup(x => x.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value, false))
+            _vacancyClient.Setup(x => x.GetVacancyApplicationsSortedAsync(vacancy.VacancyReference.Value, It.IsAny<SortColumn>(), It.IsAny<SortOrder>(), false))
```

Comment:
Can probably tighten up the setup here so it matches whats being passed through to the orchestrator... Otherwise this test will pass regardless of what value is used